src/org/wso2/tg/jenkins/executors/TestExecutor.groovy

```diff
@@ -228,3 +271,21 @@ static def getRepositoryBranch(def branch) {
         "master"
     }
 }
+
+/**
+ * Builds the bluocean console link for a given test plan
+ *
+ * @param buildURL Jenkins classic build URL
+ * @param parallelId Parallel id of the test plan
+ * @return Blueocean link
+ */
+def generateBluoceanLink(def buildURL , def parallelId){
+    def split = buildURL.split('/')
```

Comment:
URL split logics are doomed for failure.. Can you derive the url by using jenkine variables? 

See: https://testgrid-live-dev.private.wso2.com/admin/env-vars.html/

You can get jenkins url via **JENKINS_URL**. The _JOB_NAME_ provide a forward-slash separated job name.

Reply:
changed the logic to use the Jenkins env variables